packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -117,7 +117,10 @@ export class CdkToolkit {
       return this.watch(options);
     }
 
+    const startSynthTime = new Date().getTime();
     const stacks = await this.selectStacksForDeploy(options.selector, options.exclusively, options.cacheCloudAssembly);
+    const elapsedSynthTime = new Date().getTime() - startSynthTime;
+    print('\n✨  Synthesis time: %ss\n', roundPercentage(elapsedSynthTime / 1000));
```

Comment:
We seem to keep repeating this `n / 1000` expression.

Perhaps it's worth it to extract this into another helper function, same like you did for `roundPercentage()`?